exercise1.py

```diff
@@ -0,0 +1,50 @@
+# task 1
+import datetime
+import this  # appeared to be some pseudo philosophy by some dinosaur developer
+
+# task 2
+from math import sqrt
+a = 1
+b = 5
+c = 4
+
+x = (-b + sqrt(b**2 - 4*a*c))/2*a
+print(x)
+
+y = (-b - sqrt(b**2 - 4*a*c))/2*a
+print(y)
+
+# task 3
+name = input("Please enter your name: ")
+print("Hello " + name)
+
+# task 4
+exchange_rate = 36.5
+amount = input("Please enter the amount of UAH you want to exchange: ")
+uah = int(amount)
+usd = round(uah / exchange_rate, 2)
+usd_to_print = str(usd)
+print("You will receive " + usd_to_print + " USD")
+
+# task 5 aka Zavdannya 2
+snake_case = input("Please enter 3 words in format a_b_c: ")
+splited = snake_case.split("_")
+
+for word in splited:
+    cap_word = str(word.capitalize())
+    print(cap_word)
+
+# task 6 aka Zavdannya 3
+person = input("Please enter name and dates of birth and death, use * as separator: ")
+l = person.split("*")
```

Comment:
try to name variable the way it will be easy to understand what is in it, for example:
person_data_list